test/config_test/config_test.cc

```diff
@@ -168,6 +168,8 @@ uint32_t run(const std::string& directory) {
     ENVOY_LOG_MISC(info, "testing {}.\n", filename);
     OptionsImpl options(
         Envoy::Server::createTestOptionsImpl(filename, "", Network::Address::IpVersion::v6));
+    // Try to avoid conflicting with other tests.
```

Comment:
should we just disable hot restart instead? or do some of the tests rely on that?